### ISSUE TITLE
fatal error: glad/glad.h: No such file or directory

### ISSUE THREAD

**Issue by DavidJourdan** [rubenwiersma/gravo-mg#1]
Thanks for the release! I tried installing the project on Ubuntu following the instructions but compilation fails because ```glad.h``` is missing, should it be packaged with the project or do I need to install this somewhere myself?

**Comment by rubenwiersma** [comment]
Thanks for letting us know! I've been able to replicate your issue on a fresh machine. It seems to stem from libigl and one solution is to generate a glad.h file yourself at https://glad.dav1d.de (a bit more explanation and help can be found here: https://learnopengl.com/Getting-started/Creating-a-window). I'm going to check if and how we can fix this on our end. Let me know if generating the glad.h file works out for you!

**Comment by rubenwiersma** [comment]
I found a solution that doesn't involve downloading a separate `glad.h` file. The file was being called from `igl/Viewer.h`, which was imported, but not used in the gravomg library. I removed the import and was able to build it on a fresh Ubuntu machine. I committed the fix in https://github.com/rubenwiersma/gravo-mg/commit/60f5f873114e583b07c8bc55587487069ab37245.

Let me know if this solves the issue for you!
